libs/pages/elewa/about-us/src/lib/components/elewa-culture-and-values/elewa-culture-and-values.component.css

```diff
@@ -0,0 +1,229 @@
+* {
+    margin: 0;
+  }
+  
+  
+  .container {
+    margin-top: 2vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+  }
+  
+  
+  .container-section {
+    width: 100%;
+    height: 30%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+  
+  
+  .box {
+    width: 40%;
+    height: 100%;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: space-evenly;
+  }
+  
+  
+  #box-last {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: none;
+    justify-content: space-evenly;
+    margin-top: 4vh;
+  }
+  
+  
+  .container-main {
+    width: 100%;
+    height: 30%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+  
+  
+  img {
+    width: 60px;
+    height: 60px;
+  }
+  
+  
+  #box1,
+  #box3 {
+    border: thin solid grey;
+    border-left: none;
+  }
+  
+  
+  #box2,
+  #box4 {
+    border: thin solid grey;
+    border-right: none;
+  }
+  
+
+
+h2 {
+    font-size: 20px;
+    margin-left: 2vw;
+    margin-right: 2vw;
+    height: 20%;
+    font-size: var(--elewa-group-website-font-size-elewa-text);
+    font-weight: var(--elewa-group-website-font-weight-subtitle);
+  }
+  
+
+img {
+    margin-left: 2vw;
+    margin-right: 2vw;
+  }
+  
+
+p {
+    width: 80%;
+    margin-left: 2vw;
+    margin-right: 2vw;
+    height: max-content;
+    font-size: 14px;
+    text-align: justify;
+    font-family: var(--elewa-group-website-dmsans-medium);
+    font-size: var(--elewa-group-website-font-size-subheading);
+  }
+  
+  
+  .outer-p {
+    width: 30%;
+    margin-top: 10px;
+    margin-left: 8vw;
+    margin-bottom: 4vh;
+    font-family: var(--elewa-group-website-dmsans-medium);
+    font-size: var(--elewa-group-website-font-size-subheading);
+  }
+  
+  
```

Comment:
confirm on the console styles that the fonts styles are applied, they look different on your screenshot